### DISCUSSION TITLE
Updating Chaos mesh to newer version from 2.1.8+ causes chaos mesh dashboard pods to CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by Gittysri** [chaos-mesh/chaos-mesh#4284] [General]
## Bug Report

**What version of Kubernetes are you using?**
1.25.11

**What version of Chaos Mesh are you using?**
2.1.8

**What did you do? / Minimal Reproducible Example**
When Upgraded to higher version of chaos mesh 2.1.8+. Using the command "`helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing --version 2.6.2`"

**What did you expect to see?**
To see the stable version where, the pods of the chaos mesh which runs on AKS without CrashLoopBackOff Err or frequent pod restart

**What did you see instead?**
The choas mesh pods frequently gets restarted and gets into CrashLoppBackOff State

**Output of chaosctl**
Chaos version 2.1.8 was only stable in our AKS, Upgradation to chaos mesh 2.1.8+ causes the pods to restart frequently and finally gets into CrashLoopBackOff Err

**Comment by STRRL** [comment]
Could you provide the log of chaos dashboard pod for us ? thanks

**Reply by Gittysri** [reply]
While upgrading the chaos mesh from 2.1.8 to its later verion using the command "helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing --version 2.5.2". And the output is succeeded. I have provided the log of crashed chaos-mesh-dashboard pod


[chaosLog.txt](https://github.com/chaos-mesh/chaos-mesh/files/13915301/chaosLog.txt)
